nodes/cma/nodes-cma-autoscaling-custom-trigger.adoc

```diff
@@ -14,6 +14,8 @@ The custom metrics autoscaler currently supports only the Prometheus, CPU, memor
 
 You use a `ScaledObject` or `ScaledJob` custom resource to configure triggers for specific objects, as described in the sections that follow.
 
+You can xref:../../nodes/cma/nodes-cma-autoscaling-custom-trigger-auth.adoc#nodes-cma-autoscaling-custom-trigger-auth[configure a certificate authority to use with your scaled objects] or  xref:../../nodes/cma/nodes-cma-autoscaling-custom.adoc#nodes-cma-autoscaling-custom-ca_nodes-cma-autoscaling-custom[configure a custom certificate authority for all scalers in the cluster].
```

Comment:
Might be easier to see the links if this were bullets or the link for each one was highlighting a bit less text. Your call.